tests/favorites/favorites.spec.js

```diff
@@ -0,0 +1,307 @@
+var shell = require('shelljs');
+var request = require("supertest");
+var app = require('../../app');
+
+const environment = process.env.NODE_ENV || 'test';
+const configuration = require('../../knexfile')[environment];
+const database = require('knex')(configuration);
+
+describe('Favorites endpoints', () => {
```

Comment:
I'm still not sure how I feel about the large test file for each endpoint, but I suppose we should do it if it's the industry standard. Thanks for making the switch!